game-app/game-core/src/main/java/games/strategy/engine/player/Player.java

```diff
@@ -32,8 +33,7 @@ public interface Player extends IRemote {
   GamePlayer getGamePlayer();
 
   /** Called before the game starts. */
-  @RemoteActionCode(9)
-  void initialize(IPlayerBridge bridge, GamePlayer gamePlayer);
+  void initialize(PlayerBridge bridge, GamePlayer gamePlayer);
```

Comment:
This is safe. `DefaultPlayerBridge` was not serializable, so calling this over the network was not possible in the past.